package.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
I can't comment on the file directly, as it is too large, but you should not commit the package-lock.json.